projects/frontend-ui/shared/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+Change Log - VDK Shared Components Library (@vdk/shared)
+=========
```

Comment:
do we use CHANGELOG.md anywhere else in versatile-data-kit, or is it a dynamic one on new release?

Reply:
Yeah there is a CHANGELOG.md for every projects (control-service, core etc..)? 
Where does it get dynamically generated for other projects?